### PR TITLE
Add The A11Y Collective to resources

### PR DESCRIPTION
**Pull request by rianrietveld**
Hey, I kindly request you to add The A11Y Collective to the resources page.
We are an only e-learning platform for web accessibility at [a11y-collective.com](https://a11y-collective.com/). We are adding more courses and masterclasses each month, for everyone involved in the creation and maintenance of a web project.

### CODE REVIEW

File: src/_data/resources.json [hunk 1]
```diff
@@ -500,6 +500,12 @@
 			"additional": "Microsoft",
 			"url": "https://docs.microsoft.com/en-us/learn/paths/accessibility-fundamentals/"
 		},
+		{
+			"title": "An e-learning platform on Web Accessibility",
```
Comment (ericwbailey) [inline src/_data/resources.json:504]: Can we change this to `The A11Y Collective`? We use the name of the resource be the title.

File: src/_data/resources.json [hunk 2]
```diff
@@ -500,6 +500,12 @@
...
+			"description": "The A11Y Collective is the e-learning platform for anyone who wants to learn more about web accessibility. The courses focus mostly on knowledge and skills that are valuable for web designers, developers, webmasters, product owners and web editors.",
+			"additional": "The A11Y Collective",
```
Comment (ericwbailey) [inline src/_data/resources.json:506]: This is an optional field, so it can be deleted if the title is updated.

**Review by ericwbailey — CHANGES_REQUESTED** [review]

**Comment by rianrietveld** [comment]
Hey @ericwbailey, thanks for the review 🙂 
I changed the title and the additional.
Better this way?

**Review by ericwbailey — APPROVED** [review]
Looks great, thank you!

**Pull request merged**